### ISSUE TITLE
[Bug]: cn crash by index out of range [83] with length 83 when running tpcc test on eks with 2*cn

### ISSUE THREAD

**Issue by aressu1985** [matrixorigin/matrixone#8374]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93): 16acd36
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

{"level":"INFO","time":"2023/03/10 22:07:45.577223 +0000","caller":"frontend/util.go:414","msg":"rowCount 1 \ntime of getDataFromPipeline : 39.611µs \nprocessBatchTime 34.931µs \nrow2colTime 0s \nrestTime(=totalTime - row2colTime) 39.611µs \nprotoStats flushCount 0 writeCount 1 \nwriteBytes 0 0 MB\n connectionId 1025"}
{"level":"INFO","time":"2023/03/10 22:07:45.581271 +0000","caller":"frontend/util.go:414","msg":"time of Exec.Run : 104.231283ms connectionId 1021"}
{"level":"INFO","time":"2023/03/10 22:07:45.581353 +0000","caller":"frontend/util.go:414","msg":"time of SendResponse 600ns connectionId 1021"}
{"level":"INFO","time":"2023/03/10 22:07:45.581479 +0000","caller":"frontend/util.go:399","msg":"query trace status","connection_id":1021,"statement":"execute __mo_stmt_id_17 using @__mo_stmt_var_0,@__mo_stmt_var_1,@__mo_stmt_var_2,@__mo_stmt_var_3,@__mo_stmt_var_4","status":"success","span":{"trace_id":"fc6933f1-bf8f-11ed-99ac-66fd3ea098c6","kind":"statement"},"session_info":"connectionId 1021"}
panic: runtime error: index out of range [83] with length 83

goroutine 43150 [running]:
github.com/matrixorigin/matrixone/pkg/common/bitmap.(*Bitmap).AddMany(...)
	/go/src/github.com/matrixorigin/matrixone/pkg/common/bitmap/bitmap.go:198
github.com/matrixorigin/matrixone/pkg/vm/engine/tae/containers.UnmarshalToMoVec({0x2e2ed58, 0xc0376519e0})
	/go/src/github.com/matrixorigin/matrixone/pkg/vm/engine/tae/containers/utils.go:85 +0x2aa
github.com/matrixorigin/matrixone/pkg/vm/engine/tae/dataio/blockio.BlockRead({0x2dae798, 0xc0280741b0}, 0xc02d5f5020?, {0xc03293d440, 0x6, 0x6}, {0xc0101c3070, 0x6, 0x6}, {0xc02759c180, ...}, ...)
	/go/src/github.com/matrixorigin/matrixone/pkg/vm/engine/tae/dataio/blockio/read.go:68 +0x4a5
github.com/matrixorigin/matrixone/pkg/vm/engine/disttae.(*blockReader).Read(0xc011a95880, {0x2dae798?, 0xc011abee70?}, {0xc03293d440?, 0x6?, 0x6?}, 0xc024bd1ea8?, 0x41f187?)
	/go/src/github.com/matrixorigin/matrixone/pkg/vm/engine/disttae/reader.go:83 +0x399
github.com/matrixorigin/matrixone/pkg/vm/pipeline.(*Pipeline).Run(0xc01393ff28, {0x2d8d3d0, 0xc011a95880}, 0xc02ed4b800)
	/go/src/github.com/matrixorigin/matrixone/pkg/vm/pipeline/pipeline.go:70 +0x18b
github.com/matrixorigin/matrixone/pkg/sql/compile.(*Scope).Run(0xc02b4c8410, 0x0?)
	/go/src/github.com/matrixorigin/matrixone/pkg/sql/compile/scope.go:70 +0x14e
github.com/matrixorigin/matrixone/pkg/sql/compile.(*Scope).MergeRun.func1(0xc012472360?)
	/go/src/github.com/matrixorigin/matrixone/pkg/sql/compile/scope.go:90 +0x75
created by github.com/matrixorigin/matrixone/pkg/sql/compile.(*Scope).MergeRun
	/go/src/github.com/matrixorigin/matrixone/pkg/sql/compile/scope.go:85 +0x20d
sudong@sudongdeMacBook-Pro mo-cloud % kubectl -n mo-nightly-regression get pod       
[cn_crash.log](https://github.com/matrixorigin/matrixone/files/10947350/cn_crash.log)

**Comment by fengttt** [comment]
Seems vector related.   @aunjgr please take a first look.

**Comment by aunjgr** [comment]
#8395 is a quick workaround. However, setting bitmap for index which is larger than vector's length is abnormal, and should be caught in other place.

@LeftHandCold is looking into it.

**Comment by LeftHandCold** [comment]
<img width="1655" alt="image" src="https://user-images.githubusercontent.com/14086886/224627464-826a866a-b12d-4f11-9ed9-79ff6c694c79.png">
<img width="1094" alt="image" src="https://user-images.githubusercontent.com/14086886/224627578-2c049658-dd32-46d1-b0b0-922d98080326.png">
Null values are not processed when cn load data

**Comment by LeftHandCold** [comment]
same as #8186

**Comment by aunjgr** [comment]
PR has been merged

**Comment by aressu1985** [comment]
fixed:nightly-cd7e8c9
